youtube_dl/extractor/ninenow.py

```diff
@@ -86,8 +133,12 @@ def _real_extract(self, url):
                 {'geo_countries': self._GEO_COUNTRIES}),
             'id': video_id,
             'title': title,
-            'description': common_data.get('description'),
+            'description': common_data.get('episode').get('description'),
             'duration': float_or_none(video_data.get('duration'), 1000),
             'thumbnails': thumbnails,
             'ie_key': 'BrightcoveNew',
+            'season_number': season_number,
+            'episode_number': episode_number,
+            "timestamp": timestamp,
+            "upload_date": upload_date,
```

Comment:
`update_date` is pointless if `timestamp` is provided.

Reply:
I see how they are essentially the same information, but this could be useful for easier formatting when templating the output file without having to do any conversion from `timestamp` to date string.